### PR TITLE
fix(fw): EOF - Fix EXCHANGE's data_portion_length

### PR DESCRIPTION
**Pull request by pdobacz**
## 🗒️ Description

Just a minor issue I've noticed. It didn't affect any bytecode generation, but fixing anyway

## 🔗 Related Issues

NA

## ✅ Checklist
- [ ] All: Set appropriate labels for the changes.
- [ ] All: Considered squashing commits to improve commit history.
- [ ] All: Added an entry to [CHANGELOG.md](/ethereum/execution-spec-tests/blob/main/docs/CHANGELOG.md).
- [ ] All: Considered updating the online docs in the [./docs/](/ethereum/execution-spec-tests/blob/main/docs/) directory.
- [ ] Tests: All converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been added to [converted-ethereum-tests.txt](/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt).
- [ ] Tests: A PR with removal of converted JSON/YML tests from [ethereum/tests](/ethereum/tests) have been opened.
- [ ] Tests: Included the type and version of evm t8n tool used to locally execute test cases:  e.g., ref with commit hash or geth 1.13.1-stable-3f40e65.
- [ ] Tests: Ran `mkdocs serve` locally and verified the auto-generated docs for new tests in the [Test Case Reference](https://ethereum.github.io/execution-spec-tests/main/tests/) are correctly formatted.

### CODE REVIEW

File: src/ethereum_test_vm/opcode.py [hunk 1]
```diff
@@ -5066,6 +5066,7 @@ class Opcodes(Opcode, Enum):
 
     EXCHANGE = Opcode(
         0xE8,
+        data_portion_length=1,
```
Comment (pdobacz) [inline src/ethereum_test_vm/opcode.py:5069]: late thought - one could also derive it dynamically as `len(data_portion_formatter(args))`, please advise if this is better. This should cover `RJUMPV` as well then.

**Comment by danceratopz** [comment]
Hi @pdobacz, the initial CI fail have been fixed by #850 so I merged `main` into this branch. Sorry about that.

File: src/ethereum_test_vm/opcode.py (hunk 1 above)
Comment (marioevz) [reply src/ethereum_test_vm/opcode.py:5069]: I think this should be fine as is by now :+1:

**Review by marioevz — APPROVED** [review]
LGTM, thanks!

**Pull request merged**